exts/cesium.omniverse/cesium/omniverse/ui/attributes/data_attributes_widget.py

```diff
@@ -23,5 +23,11 @@ def _customize_props_layout(self, props):
                 CustomLayoutProperty("cesium:georeferenceOrigin:latitude", "Latitude")
                 CustomLayoutProperty("cesium:georeferenceOrigin:longitude", "Longitude")
                 CustomLayoutProperty("cesium:georeferenceOrigin:height", "Height")
+            with CustomLayoutGroup("Debug Options"):
```

Comment:
Let's start this collapsed. Just add `collapsed=True` to the arguments for `CustomLayoutGroup`.